lib/dialects/cockroachdb.ts

```diff
@@ -0,0 +1,155 @@
+import { Column } from '../types/column';
+import { parseDefaultValue as postgresParseDefaultValue, default as Postgres } from './postgres';
+
+type RawColumn = {
```

Comment:
is this different from pg one?

Reply:
yes, as it converts string to number - cockroach driver returns strings instead of numbers (as they are 64bit integers)